stepler/nova/steps/servers.py

```diff
@@ -113,6 +113,7 @@ def predicate():
     def check_ssh_connect(self, server, keypair, username=None, password=None,
                           ip=None, proxy_cmd=None, ssh_timeout=60, timeout=0):
         """Verify step to check ssh connect to server."""
+        server.get()
```

Comment:
@AlexanderKoryagin, pls explain, why we should refresh server before each action. Seems, we must pass the fresh server object to steps.